File: sdk/typescript/rooch-sdk/src/types/bytes.ts
// Copyright (c) RoochNetwork
// SPDX-License-Identifier: Apache-2.0

export type Bytes = Uint8Array

export const EmptyBytes = new Uint8Array()

/**
 * Shortcut to one-element (element is 0) byte array
 */
export const NullBytes = new Uint8Array([0])

File: sdk/go/rooch-sdk/types/bytes.go
// Copyright (c) RoochNetwork
// SPDX-License-Identifier: Apache-2.0

package types

// Bytes is a type alias for []byte
type Bytes []byte

// EmptyBytes represents an empty byte slice
var EmptyBytes = make([]byte, 0)

// NullBytes represents a one-element byte slice containing zero
var NullBytes = []byte{0} 